### DISCUSSION TITLE
Hidden mod should affect spinners

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#14032] [Ideas] [answered]
Spinners are currently not hidden by the HD mod. Maybe show them for a short time before hiding?

**Comment by smoogipoo** [comment, marked as answer]
I think it's just lacking implementation?

Edit: Nope, it currently looks exactly the same as stable. Interesting.

**Reply by peppy** [reply]
Is this asking for spinners to hide as a proposal? I don't believe they do in stable, correct?

**Reply by smoogipoo** [reply]
Yeah it looks exactly the same as in stable, which doesn't hide. I'm obviously not an HD player :sweat_smile:

**Reply by Moltanicaa** [reply]
This is proposal

**Reply by peppy** [reply]
I'm not against it hiding. It probably wasn't done so on stable for some logistical reason.

**Comment by NafeeJ** [comment]
HD does affect spinners though? the "completion circle", which shows you how long until the spinner is done (then you get bonus score for spinning), is gone with HD. Many skins remove the completion circle so maybe thats why you see no difference?

**Reply by peppy** [reply]
The proposal is to hide the whole spinner, as is done with other objects.
